# Data Collection

In this notebook, we demonstrate an expample of data collection. The data consists of food recipes (cuisines and ingredientes)

In [1]:
import pandas as pd
# URL donde estan los datos almacenados
# En este caso, en un archivo tipo .csv
path = "https://cf-courses-data.s3.us.cloud-object-storage.appdomain.cloud/IBMDeveloperSkillsNetwork-" \
        "DS0103EN-SkillsNetwork/labs/Module%202/recipes.csv"
# Cargar datos
recipes = pd.read_csv(path)

In [3]:
# Show the first few rows
recipes.head()

,country,almond,angelica,anise,anise_seed,apple,apple_brandy,apricot,armagnac,artemisia,...,whiskey,white_bread,white_wine,whole_grain_wheat_flour,wine,wood,yam,yeast,yogurt,zucchini
0,Vietnamese,No,No,No,No,No,No,No,No,No,...,No,No,No,No,No,No,No,No,No,No
1,Vietnamese,No,No,No,No,No,No,No,No,No,...,No,No,No,No,No,No,No,No,No,No
2,Vietnamese,No,No,No,No,No,No,No,No,No,...,No,No,No,No,No,No,No,No,No,No
3,Vietnamese,No,No,No,No,No,No,No,No,No,...,No,No,No,No,No,No,No,No,No,No
4,Vietnamese,No,No,No,No,No,No,No,No,No,...,No,No,No,No,No,No,No,No,No,No


In [4]:
# Get the dimensions of the dataframe
recipes.shape

(57691, 384)

So our dataset consists of **57,691 recipes**. Each row represents a recipe, and for each recipe, the corresponding cuisine is documented as well as whether **384 ingredients** exist in the recipe or not, beggining with almond and ending with zucchini. Ingredients are listed in alphabetical order.